### PR TITLE
feat: add Uno hacktoberfest swag listing

### PR DESCRIPTION
**Pull request by kev-odin**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

File: docs/list.md [hunk 1]
```diff
@@ -332,6 +332,23 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
 - **Notes**: Check out the [blog post](https://blog.flyte.org/meet-flyte-and-unionml-at-hacktoberfest-2022){:target="_blank"} for more info.
 
+#### **Uno**
+
+- **Swag**: Hoodie, T-shirt, eBook
+- **Requirements**:
+    - Submit 2 Quality Pull Requests for Getting Started with the Uno Platform and WinUI 3 eBook
+    - Submit 4 Quality Pull Requests for a Uno Platform T-shirt
+    - Submit 10 Quality Pull Requests for a Uno Platform Hoodie
+- **How to sign up**:
```
[the remark this answers is not in the document]
Comment (kev-odin) [reply docs/list.md]: Removed trailing spaces

File: docs/list.md [hunk 2]
```diff
@@ -332,6 +332,23 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+#### **Uno**
+
+- **Swag**: Hoodie, T-shirt, eBook
+- **Requirements**:
+    - Submit 2 Quality Pull Requests for Getting Started with the Uno Platform and WinUI 3 eBook
+    - Submit 4 Quality Pull Requests for a Uno Platform T-shirt
+    - Submit 10 Quality Pull Requests for a Uno Platform Hoodie
+- **How to sign up**: 
+    - Register on the [Hacktoberfest website](https://hacktoberfest.com/){:target="\_blank"} anytime between September 26 and October 31, 2022.
+- **Issues**:
```
[the remark this answers is not in the document]
Comment (kev-odin) [reply docs/list.md]: Removed trailing spaces

File: docs/list.md [hunk 3]
```diff
@@ -332,6 +332,23 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+    - Submit 4 Quality Pull Requests for a Uno Platform T-shirt
+    - Submit 10 Quality Pull Requests for a Uno Platform Hoodie
+- **How to sign up**: 
+    - Register on the [Hacktoberfest website](https://hacktoberfest.com/){:target="\_blank"} anytime between September 26 and October 31, 2022.
+- **Issues**: 
+    - List of [hacktoberfest issues](https://github.com/unoplatform/uno/labels/hacktoberfest){:target="_blank"}
+    - List of [open issues](https://github.com/unoplatform/uno/issues){:target="_blank"}
+    - List of [documentation contributions](https://github.com/orgs/unoplatform/projects/77/views/1){:target="_blank"}
+    - List of [advocacy contributions](https://github.com/unoplatform/uno/blob/master/doc/articles/guides/community-tutorials.md){:target="_blank"}
+- **Notes**:
```
[the remark this answers is not in the document]
Comment (kev-odin) [reply docs/list.md]: Removed trailing spaces

**Comment by crweiner** [comment]
Hey @kev-odin! 👋 Thank you for adding Uno to the Hacktoberfest Swag List! I appreciate you making sure to fix the md flags that CodeClimate brought up right away and for taking the time to make a through entry! I made a few changes and will get it merged. 👍 

Thank you for helping to make the Swag List better, and I hope we can send lots of good PRs Uno's way!

thank you

Have a great Hacktoberfest! 🎃

**Pull request merged**